<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_entrar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #DA2228;
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_caminho_entrada">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Insira todo o caminho onde a pasta se encontra: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_barra_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>-40</y>
     <width>20</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagens/faixa.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_logo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagens/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_entrada_del_pasta">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_titulo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Deletar Pastas - COMPARTILHADO</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_barra">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-40</y>
     <width>20</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imagens/faixa.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_explorador_arquivo">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: blue;
color: white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Abrir explorador de arquivos</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_obs">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>721</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>*OBS: Informe o caminho até dentro da pasta que deseja excluír. </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_obs_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Ex:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_obs_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string> C:\teste\pasta_excluir (A pasta que desejo excluír é a &quot;pasta_excluir&quot;)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_entrada_del_pasta</tabstop>
  <tabstop>btn_entrar</tabstop>
  <tabstop>btn_explorador_arquivo</tabstop>
  <tabstop>btn_voltar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
